packages/contracts-bedrock/src/dispute/PermissionedDisputeGame.sol

```diff
@@ -100,4 +100,18 @@ contract PermissionedDisputeGame is FaultDisputeGame {
         // Fallthrough initialization.
         super.initialize();
     }
+
+    ////////////////////////////////////////////////////////////////
+    //                     IMMUTABLE GETTERS                      //
+    ////////////////////////////////////////////////////////////////
+
+    /// @notice Returns the proposer address.
```

Comment:
Looks great. Can we assert that these are set properly in `FPACOPS.s.sol` & `PermissionedDisputeGame.t.sol`, now that we have accessors?